frontend/src/data/emails.js

```diff
@@ -1,63 +1,157 @@
 const EMAILS = [
   {
+    id: '0',
     subject: 'Welcome on board!',
-    from: {
-      name: 'Arther Ardent',
-      photoUrl: 'https://randomuser.me/api/portraits/men/32.jpg',
-    },
-    id: 'fa59b235-184f-40ba-aae7-daaf48689d22',
-    timestamp: '2018-02-20:15:31.000Z',
-    body: `Hello Ana,\n\nThank you for helping out with the conference organisation. Great to have you on board!\n\nWe are currently...`,
+    from: 'Arther Ardent',
+    emails: [
+      {
+        subject: 'Welcome on board!',
+        from: {
+          name: 'Arther Ardent',
+          photoUrl: 'https://randomuser.me/api/portraits/men/32.jpg',
+        },
+        id: 'fa59b235-184f-40ba-aae7-daaf48689d22',
+        timestamp: '2018-02-20:15:31.000Z',
+        body: `Hello Ana,\n\nThank you for helping out with the conference organisation. Great to have you on board!\n\nWe are currently...`,
+      },
+    ],
```

Comment:
This data structure doesn't follow how the backend is modeling things. A more accurate model would be:

```javascript
type Email = {